### DISCUSSION TITLE
About `uvicorn[standard]`

### DISCUSSION THREAD

**Discussion by abersheeran** [Kludex/uvicorn#1274] [Q&A]
When deploying, only uvloop and httptools are often needed, and no additional dependencies such as watchgod and python-dotenv are needed.

Can we add one, such as `uvicorn[fast]`?

**Comment by euri10** [comment]
you can see https://github.com/encode/uvicorn/pull/666 and the issue it closes, it's been discussed a lot, I dont say I'm against, just not sure I want to go back on this topic, unless there's a compelling case

**Reply by abersheeran** [reply]
Thanks. I will read them.
